libs/components/popovers/testing/src/dropdown/popovers-fixture-dropdown-item.ts

```diff
@@ -1,3 +1,6 @@
+/**
+ * @internal
+ */
 export interface SkyPopoversFixtureDropdownItem {
   ariaRole: string;
 }
```

Comment:
this was showing up in the testing tab of popovers currently, adding @internal to prevent that